### ISSUE TITLE
Support for The Things Network Gateway Connector

### ISSUE THREAD

**Issue by alviso** [ttn-zh/ic880a-gateway#12]
Can you guys update the script to support the Gateway Connector according to this
https://www.thethingsnetwork.org/docs/gateways/registration.html
Thanks, Peter

**Comment by gonzalocasas** [comment]
@alviso: yes, we will definitely do so as soon as possible. But it means updating the packet forwarder with a version that uses the new TTN connector, so, it's not a quickie.

**Comment by alviso** [comment]
@gonzalocasas Thanks, looking forward to it!

**Comment by jpmeijers** [comment]
A beta version of the new gateway connector for Linux is now available at:
https://github.com/kersing/packet_forwarder/tree/master/mp_pkt_fwd

It will be great if you could update you repository to support this. Because after that we can also solve this issue:
https://github.com/rayozzie/ttn-resin-gateway-rpi/issues/2

**Comment by cimm** [comment]
The Things Network [packet forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/master) seems to be put on hold. What is the recommended package forwarder to use now, the Semtech UDP one until future notice?

**Comment by gonzalocasas** [comment]
You have two options right now: either the Semtech UDP one, or the [mp packet forwarder](https://github.com/kersing/packet_forwarder/tree/master/mp_pkt_fwd). 

In the future, the gateway connector will be used by the gateway agent to interact with the Semtech UDP package forwarder (once v3 is released).
